update.sh

```diff
@@ -13,12 +13,16 @@ versions=( "${versions[@]%/}" )
 template=
 dockerfile=
 
+yarnVersion="$(curl -sSL --compressed https://api.github.com/repos/yarnpkg/yarn/releases/latest | grep '"tag_name":' | head -n 1 | cut -d '"' -f 4 | cut -d v -f 2)"
```

Comment:
You can use https://yarnpkg.com/latest-version to get the latest version number.